syncd/FlexCounter.cpp

```diff
@@ -511,8 +519,72 @@ class CounterContext : public BaseCounterContext
         }
     }
 
+    void addVoqEntryToChassisAppDb(_In_ const sai_object_id_t rid,
+                                   _In_ const sai_object_id_t vid,
+                                   _In_ std::vector<swss::FieldValueTuple> &counterValues,
+                                   _Out_ swss::Table *appDbCountersTable ) {
+       // We need to update CHASSIS_APP_DB only if it is a VOQ counter. So first check
+       // if the counter is a QUEUE counter and if yes, query the queue attribute to check
+       // if it is a VOQ queue or not.
+       sai_status_t status;
+
+       if (m_objectType == SAI_OBJECT_TYPE_QUEUE) {
+          sai_attribute_t attr;
+          attr.id = SAI_QUEUE_ATTR_TYPE;
+          status = sai_queue_api->get_queue_attribute(rid, 1, &attr);
+          if (status != SAI_STATUS_SUCCESS ) {
+             SWSS_LOG_ERROR( "sai get_queue_attribute failed for SAI_QUEUE_ATTR_TYPE. "
+                   "status %d rid 0x%" PRIx64 "vid 0x%" PRIx64, status, rid, vid);
+             return;
+          }
+          if (attr.value.s32 == SAI_QUEUE_TYPE_UNICAST_VOQ) {
+             attr.id = SAI_QUEUE_ATTR_INDEX;
+             status = sai_queue_api->get_queue_attribute(rid, 1, &attr);
+             if (status != SAI_STATUS_SUCCESS ) {
+                SWSS_LOG_ERROR( "sai get_queue_attribute failed for SAI_QUEUE_ATTR_INDEX."
+                      " status %d rid 0x%" PRIx64 "vid 0x%" PRIx64, status, rid, vid) ;
+                return;
+             }
+             auto iface = counterDb.hget(COUNTERS_PORT_NAME_VOQ_MAP, sai_serialize_object_id(vid));
+             if (!iface)
+             {
+                SWSS_LOG_ERROR("Cannot query port name for queue. rid 0x%" PRIx64 "vid 0x%" PRIx64, rid, vid);
+                return;
+             }
```

Comment:
add empty line before each if and after each }

Reply:
Done